src/app/functions/server/getRelevantGuidedSteps.ts

```diff
@@ -206,8 +218,11 @@ function hasCompleted(data: InputData, stepId: StepLink["id"]): boolean {
     dataClass: (typeof BreachDataTypes)[keyof typeof BreachDataTypes]
   ): boolean {
     return !data.subscriberBreaches.some((breach) => {
+      const effectedDataClasses = breach.dataClassesEffected.map(
```

Comment:
Heh OK, there's already a spelling error in the breach property, but also here:

```suggestion
      const affectedDataClasses = breach.dataClassesEffected.map(
```

Reply:
Oh. Updated in [b51cce1](https://github.com/mozilla/blurts-server/pull/3493/commits/b51cce1202b0bc5f905da2e0cf06915eba41e938).